source/_integrations/shelly.markdown

```diff
@@ -40,6 +40,22 @@ Integration is communicating directly with the device; cloud connection is not n
 
 {% include integrations/config_flow.md %}
 
+{% configuration %}
```

Comment:
This needs additional explanation, maybe even its own chapter.

What is it? what does it do? Why do I need this? Do I need this?

Reply:
Was my original idea, but then I was asked to move to to current view.

Will update later

Reply:
Done, looking forward for your feedback